src/olympia/reviewers/tests/test_decorators.py

```diff
@@ -34,16 +32,18 @@ def test_301_by_slug(self):
         self.assert3xx(res, self.id_path, 301)
 
     def test_301_by_guid(self):
-        self.request.path = f'http://testserver/addon/{quote(self.addon.guid)}/reviews'
+        self.request.path = f'http://testserver/addon/{self.addon.guid}/reviews'
         res = self.view(self.request, self.addon.guid)
         self.assert3xx(res, self.id_path, 301)
 
     def test_slug_replace_no_conflict(self):
-        slug = quote(self.addon.slug.encode('utf8'))
+        slug = self.addon.slug
         self.request.path = f'http://testserver/addon/{slug}/reviews/{slug}/path'
 
         res = self.view(self.request, self.addon.slug)
-        redirection = f'http://testserver/addon/{self.addon.id}/reviews/{slug}/path'
+        redirection = (
+            f'http://testserver/addon/{self.addon.id}/reviews/{quote(slug)}/path'
```

Comment:
why is it quoted here?

Reply:
We receive the unquoted path but we quote it (via `urlencode()`) when returning the new URL (the redirection)